README.md

```diff
@@ -77,6 +77,36 @@ If you want to create a package with all the dependencies bundled, you can run `
 
 Every patch is expected to adhere to our coding style, which is checked automatically by Travis. You can install the checkers locally using `npm run install-dependencies`, and then run the checks using `npm run check` before submitting a pull request. There is also `npm run fix`, that will attempt to fix the formatting.
 
+## Dockerizing selfoss
```

Comment:
The readme is also too cluttered. I would move this section to `docs/docker.md` and link it from the `README`.

Reply:
I moved it to utils/docker, so that it gets displayed to anyone lurking in that directory.